.eslintrc

```diff
@@ -26,5 +19,36 @@ rules:
   # enforce comma at eol (never before)
   comma-style: [2, "last"]
 
+  # 2 spaces indentation
+  indent: [2, 2]
+
+  # quotes are always double
+  quotes: [2, "double"]
+
+  # single empty lines only
+  no-multiple-empty-lines: [2, {"max": 1}]
+
   # allow functions declarations at bottom of scope
   no-use-before-define: [2, "nofunc"]
+
+  # semicolons are useless
+  semi: [2, "never"]
+
+  # consistent whitespace after keywords
+  space-after-keywords: [2, "always"]
+
+  # consistent whitespace before blocks
+  space-before-blocks: [2, "always"]
+
+  # consistent whitespace before function paren
+  space-before-function-paren: [2, {"anonymous": "always", "named": "never"}]
+
+  # consistent whitespace in brackets
+  # disabled as possibly buggy: https://github.com/babel/babel-eslint/issues/87
```

Comment:
Ok I tested and this is fixed in eslint master. You guys just got to wait for the next release or change the `package.json` to reference that commit.